include/rcutils/error_handling.h

```diff
@@ -41,7 +43,13 @@ typedef struct rcutils_error_state_t
   rcutils_allocator_t allocator;
 } rcutils_error_state_t;
 
-#define RCUTILS_SAFE_FWRITE_TO_STDERR(msg) fwrite(msg, sizeof(char), sizeof(msg), stderr)
+#ifdef __STDC_LIB_EXT1__ || WIN32
```

Comment:
please use `_WIN32` instead of `WIN32` like the rest of the codebase

Reply:
Does this comparison ever evaluate to false ?

As it stands `__STDC_LIB_EXT1__` is always defined isn't it ? or is `string.h` undefining it ?

Reply:
Note: @dhood is defining `__STDC_WANT_LIB_EXT1__` above not `__STDC_LIB_EXT1__`, so my guess is `string.h` might set it.

Reply:
I see, it may deserve a comment to explain what's going on here

Reply:
I didnt know that modern GCC now accept boolean conditions on `ifdef` I remember always having to use `#if defined` for this kind of operation

Reply:
Yeah, that is odd, I wouldn't be surprised if that didn't do what your expecting.